config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -343,7 +343,7 @@ periodics:
   annotations:
     testgrid-dashboards: provider-azure-cloud-provider-azure
     testgrid-tab-name: cloud-provider-azure-master
-    testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
+    testgrid-alert-email: kubernetes-provider-azure-oot@googlegroups.com
```

Comment:
why do we change this? seems I'm not part of it

Reply:
Could you try joining it here: https://groups.google.com/g/kubernetes-provider-azure-oot?
We wanted to separate testgrid alerts between k/k and out-of-tree components.

Reply:
there's no join options on the page